_docs/15c-permissions-example.md

```diff
@@ -210,3 +256,158 @@ export const SERVICE_CONFIG: Object = {
 };
 ```
 
+## Route permissions
+
+If we want to manage the routes permissions we need to define on the `main-routing.module.ts` the `canActivateChild` parameter.
+
+`main-routing.module.ts`:
+```javascript
+...
+export const routes: Routes = [
+  {
+    path: '',
+    component: MainComponent,
+    canActivate: [AuthGuardService],
+    canActivateChild: [PermissionsGuardService],
+    children: [
+...
+```
+
+The last configuration that we can do it's to define a redirection page to visit when the user does not have permissions to access the page requested. To configure this page you need to add the route (or use the ontimize component `403`) to the `restrictedPermissionsRedirect` parameter on the routing module of the component. Example:
+
+`candidates-routing.module.ts`:
+```javascript
+...
```

Comment:
In this example, it would be a good idea to write an example with restricted permission on a specific module and screen as CandidatesNewComponent